autoload/slime.vim

```diff
@@ -138,7 +138,14 @@ endfunction
 function! s:WeztermConfig() abort
   if !exists("b:slime_config")
     let b:slime_config = {"pane_id": 1}
-  end
+  elseif exists("b:slime_config.pane_direction")
+    let tmp = system("wezterm cli get-pane-direction " . shellescape(b:slime_config["pane_direction"]), "\n")
+    if tmp != ""
+      let b:slime_config = {"pane_id": tmp}
+    else
+      let b:slime_config = {"pane_id": 1}
+    endif
+  endif
```

Comment:
after line 140?

```vim
if exists("b:slime_config.pane_direction")
  let pane_id = system("wezterm cli get-pane-direction " . shellescape(b:slime_config["pane_direction"]))
  let b:slime_config = {"pane_id": pane_id}
endif
```

equivalent logic?

Reply:
My thought was if, e.g., we are in the rightmost pane and ask for the pane to the right, then we get an empty return and it would be nice to have a default. Though I guess this is just the suggestion for the user, so we can just leave it empty in that case. I'll update.